src/discord/events/message/supportRequest.ts

```diff
@@ -1,32 +1,78 @@
 import Discord from 'discord.js';
-import { SupportRequest, SupportRequestType, SupportRequestErrors } from '../../../entities/supportRequest';
+import { Not } from 'typeorm';
+import { SupportRequest, SupportRequestType, SupportRequestStatus } from '../../../entities/supportRequest';
 import logger from '../../../logger';
 import { Config } from '../../../entities/config';
+import { SubCommands } from '.';
+import { DiscordContext } from '../../../entities/discordContext';
 
-export async function supportRequest(msg: Discord.Message): Promise<void> {
-  const discordId = msg.author.id;
-  const discordName = msg.author.username;
-  const actionId = msg.content === '!technicalSupport' ? SupportRequestType.TechnicalSupport : SupportRequestType.IdeaPitch;
+/* eslint-disable no-param-reassign */
 
+interface UserInfo {
```

Comment:
Looks like this is being used for the payload, can we rename this to `PayloadInfo` so it's clear what the usage is?